packages/rest/src/cliAgent.ts

```diff
@@ -61,7 +93,37 @@ export async function runRestAgent(restConfig: AriesRestConfig) {
     logger,
   }
 
-  const agent = new Agent(agentConfig, agentDependencies)
+  const agent = new Agent({
+    config: agentConfig,
+    dependencies: agentDependencies,
+    modules: {
+      connections: new ConnectionsModule({
+        autoAcceptConnections,
+      }),
+      proofs: new ProofsModule({
+        autoAcceptProofs,
+      }),
+      credentials: new CredentialsModule({
+        autoAcceptCredentials,
+      }),
```

Comment:
As you may have noticed, Proofs and Credentials modules support by default only V2 protocols. And they don't set any specific credential/proof format, meaning that they are not actually usable.

I think it will be better to create a configuration where the agent supports both V1 and V2 protocols for each, using legacy Indy credentials and AnonCreds. Look at [`getAskarAnonCredsIndyModules`](https://github.com/hyperledger/aries-framework-javascript/blob/4856b8f6632a4d10faeb1b156397226b64e3b8b6/demo/src/BaseAgent.ts#L110) from AFJ demo for a good example about what I'm referring to.

Reply:
> As you may have noticed, Proofs and Credentials modules support by default only V2 protocols. And they don't set any specific credential/proof format, meaning that they are not actually usable.

We should change this...